index.html

```diff
@@ -3484,6 +3484,29 @@ <h3>Media Types</h3>
 those other syntaxes to be transformed to the `application/vc+ld+json`
 media type.
         </p>
+
+        <p>
+Additional `application/vc+` media type transformation specifications can be
+defined to provide implementers with further options for producing objects
+compliant with the base media type.
+
+Specifications are registered with the [[?VC-SPECS-DIR]] and
+MUST result in a serialization
+that is objectively testable via a test suite.
+
+Transformations from other media types MUST state the directionallity of the 
+transformation from the options:
+
+<ul>
+  <li>Unidirectional</li>
+  <li>Bidirectional</li>
+</ul>
+
+Bidirectional transformations MUST result in a lossless conversion between 
+the originating data format and the resulting object compliant with the base 
+media type.
```

Comment:
```suggestion
Bidirectional transformations MUST result in a lossless conversion between 
the originating serialization and a resulting `application/vc+ld+json` serialization 
that conforms to the data model in this specification.
```

Reply:
That still reads as unidirectional to me. I've added `round-trip` to make it bidirectional.
```suggestion
Bidirectional transformations MUST result in a lossless round-trip conversion between 
the originating serialization and a resulting `application/vc+ld+json` serialization 
that conforms to the data model in this specification.
```

Reply:
Byte equivalence is certainly one way to do it, but I worry it might be unnecessarily restrictive for some formats. I could imagine that the byte order could matter for some formats and not others. Maybe we want language like "functionally equivalent"?